src/Dao/UserDaoImplement.java

```diff
@@ -0,0 +1,87 @@
+package Dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import util.Constans;
+import util.SqlConnection;
+import welldevdemomodel.io.User;
+
+public class UserDaoImplement implements UserDao{
+
+
+	@Override
+	public boolean insertUser(User user) {
+		try {
+		Connection connection= SqlConnection.getInstance().getConnection();
```

Comment:
Declaring same connection for every method inside the class. You could declare this as object variable.